src/pages/resultOfSearch.tsx

```diff
@@ -0,0 +1,218 @@
+          <RersultCard
+            image={<img src={astronaut} alt="astronaut"></img>}
+            text={<div>{BigCardItem.map(({ text }) => text)}</div>}
+            date={<div>{BigCardItem.map(({ date }) => date)}</div>}
+            title={<div>{BigCardItem.map(({ title }) => title)}</div>}
+          ></RersultCard>
+          <RersultCard
+            image={<img src={astronaut} alt="astronaut"></img>}
+            text={<div>{BigCardItem.map(({ text }) => text)}</div>}
+            date={<div>{BigCardItem.map(({ date }) => date)}</div>}
+            title={<div>{BigCardItem.map(({ title }) => title)}</div>}
+          ></RersultCard>
+          <RersultCard
+            image={<img src={astronaut} alt="astronaut"></img>}
+            text={<div>{BigCardItem.map(({ text }) => text)}</div>}
+            date={<div>{BigCardItem.map(({ date }) => date)}</div>}
+            title={<div>{BigCardItem.map(({ title }) => title)}</div>}
+          ></RersultCard>
+        </ResultOfCardBody>
+
+        <ResultOfSearchOtherPages>
+          <ResultOfSearchLeftArrow>
+            <LefrArrowBtn>
+              <img src={LeftArrow} alt="prev"></img>
+              <LeftArrowText>Prev</LeftArrowText>
+            </LefrArrowBtn>
+          </ResultOfSearchLeftArrow>
+          <OtherPage>
+            <PageOne>
+              <PageOneLink href="#">1</PageOneLink>
+            </PageOne>
+            <PageTwo>
+              <PageTwoLink href="#">2</PageTwoLink>
+            </PageTwo>
+            <PageThree>
+              <PageThreeLink href="#">3</PageThreeLink>
+            </PageThree>
+            <PageFour>
+              <PageFourLink href="#">...</PageFourLink>
+            </PageFour>
+            <PageEight>
+              <PageEightLink href="#">8</PageEightLink>
+            </PageEight>
+          </OtherPage>
+          <ResultOfSearchRightArrow>
+            <RightArrowBtn>
+              <RightArrowText>Next</RightArrowText>
+              <img src={RightArrow} alt="next"></img>
+            </RightArrowBtn>
+          </ResultOfSearchRightArrow>
+        </ResultOfSearchOtherPages>
+
+        <FooterYear>
+          <FooterTextWrapper>
+            <Year>{new Date().getFullYear()}</Year>
+            <Rights>All rights reserved</Rights>
+          </FooterTextWrapper>
+        </FooterYear>
+      </ResultOfSearchWrapper>
+    </ResultOfSearchBg>
+  );
+};
+
+const ResultOfSearchBg = styled.div`
+  width: 100%;
+  min-height: 100vh;
+  background-color: var(--background-primary-color);
+`;
+
+const ResultOfSearchWrapper = styled.div`
+  width: 930px;
+  margin: auto;
+`;
+
+const ResultTitle = styled.h1`
+  color: var(--text-primary-color);
+  margin: 50px auto;
+`;
+
+const ResultOfCardBody = styled.div``;
+
+const ResultOfSearchOtherPages = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 0 30px;
+`;
+
+const ResultOfSearchLeftArrow = styled.div`
+  display: flex;
+  align-items: center;
+
+  img {
```

Comment:
better to not use nested declarations where styled components can be used;

use it only if you have logic related with parent node